website/docs/d/order_cart_product_options_plan.html.markdown

```diff
@@ -20,13 +20,14 @@ data "ovh_order_cart_product_options_plan" "plan" {
   product           = "cloud"
   plan_code         = "project"
   options_plan_code = "vrack"
+  catalog_name      = "iplb_private_beta"
 }
 ```
 
 ## Argument Reference
 
 * `cart_id` - (Required) Cart identifier
-* `catalog_name` - Catalog name
+* `catalog_name` - Catalog name (Internal API - restricted to OVHcloud internal employees)
```

Comment:
It is not acceptable to add parameters that are restricted to OVHcloud employees on the public provider. You should check internally with the team responsible for these routes if it's possible to make this parameter public and then we will check this PR.